### PR TITLE
2022EE samples

### PR DESCRIPTION
**Pull request by ahmad3213**

### CODE REVIEW

File: config/HH_bbWW/Run3_2022EE/samples.yaml [hunk 1]
```diff
@@ -0,0 +1,640 @@
...
+Muon_Run2022F:
+  sampleType: data
+Muon_Run2022G:
+  sampleType: data
+MuonEG_Run2022E:
+  sampleType: data
+MuonEG_Run2022F:
+  sampleType: data
+MuonEG_Run2022G:
+  sampleType: data
```
Comment (kandrosov) [inline config/HH_bbWW/Run3_2022EE/samples.yaml:640]: - data datasets should be defined in `config/Run3_2022EE/samples.yaml`
- define all datasets that we can potentially use in bbWW or bbtautau analyses

File: config/Run3_2022/samples.yaml [hunk 2]
```diff
@@ -1,17 +1,23 @@
...
   - 'T2_CH_CERN:/store/group/phys_higgs/HLepRare/skim_2024_v1/Run3_2022'
   nano_version: v14
   crossSectionsFile : config/crossSections13TeV.yaml
   use_stitching: false
+GluGlutoRadiontoHHto2B2Vto2B2JLNu_M-250:
+   sampleType: GluGluToRadion
+   mass: 250
+   spin: 0
+   crossSection: 1pb
+   generator: madgraph
```
Comment (kandrosov) [inline config/Run3_2022/samples.yaml:14]:
```suggestion
```

File: config/Run3_2022EE/samples.yaml [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+  - 'T2_CH_CERN:/store/group/phys_higgs/HLepRare/skim_2024_v1/Run3_2022EE'
+  nano_version: v14
+  crossSectionsFile : config/crossSections13TeV.yaml
+  use_stitching: false
+GluGlutoRadiontoHHto2B2Vto2B2JLNu_M-250:
+   sampleType: GluGluToRadion
+   mass: 250
+   spin: 0
+   crossSection: 1pb
+   generator: madgraph
```
Comment (kandrosov) [inline config/Run3_2022EE/samples.yaml:14]:
```suggestion
```

File: config/HH_bbWW/Run3_2022/samples.yaml [hunk 4]
```diff
@@ -618,4 +618,21 @@ VBFHHto2B2Vto2L2Nu_CV-m2p12_C2V-3p87_C3-m5p96:
...
+Muon_Run2022D:
+  sampleType: data  
+SingleMuon_Run2022C:
+  sampleType: data  
+DoubleMuon_Run2022C:
+  sampleType: data  
+MuonEG_Run2022C:
+  sampleType: data  
+MuonEG_Run2022D:
+  sampleType: data
```
Comment (kandrosov) [inline config/HH_bbWW/Run3_2022/samples.yaml:638]: - data datasets should be defined in `config/Run3_2022/samples.yaml`
- define all datasets that we can potentially use in bbWW or bbtautau analyses

File: config/HH_bbWW/Run3_2022/samples.yaml [hunk 5]
```diff
@@ -618,4 +618,21 @@ VBFHHto2B2Vto2L2Nu_CV-m2p12_C2V-3p87_C3-m5p96:
   sampleType: HHnonRes
   generator: madgraph
   #crossSection: To_be_added
-############## Finished non -Resonant Signal samples)##############
```
Comment (kandrosov) [inline config/HH_bbWW/Run3_2022/samples.yaml:621]: Remove trailing whitespaces in all modified files

File: config/HH_bbWW/Run3_2022EE/samples.yaml [hunk 6]
```diff
@@ -0,0 +1,640 @@
+############## Start Resonant Signal samples)##############
```
Comment (kandrosov) [inline config/HH_bbWW/Run3_2022EE/samples.yaml:1]: Missing samples:
- VBFHHto2B2V_CV-m1p83_C2V-3p57_C3-m3p39
- VBFHHto2B2Vto2L2Nu_CV-m1p83_C2V-3p57_C3-m3p39

Missing generator:
- GluGlutoRadiontoHHto2B2Vto2B2JLNu_M-1800
- GluGlutoRadiontoHHto2B2Vto2B2JLNu_M-3000
- GluGlutoHHto2B2V_kl-1p00_kt-1p00_c2-0p00

Set cross-section for all samples (to allow running the code). You can use 1pb placeholder for nonres samples

File: config/Run3_2022/samples.yaml [hunk 7]
```diff
@@ -1,17 +1,23 @@
 GLOBAL:
   era: Run3_2022
-  luminosity: 1000. # fb-1
+  luminosity: 7980.4 # fb-1
```
Comment (kandrosov) [inline config/Run3_2022/samples.yaml:3]:
```suggestion
  luminosity: 7980.4 # pb-1
```

**Review by kandrosov — CHANGES_REQUESTED** [review]

**Review by kandrosov — CHANGES_REQUESTED** [review]
@ahmad3213 thank you! All looks good to me. The only remaining issue before merging the PR are trailing whitespaces in multiple lines...

**Comment by ahmad3213** [comment]
i do not think there should be whitespaces now. I understand why this was happening, because vi editor was getting crazy.

**Review by kandrosov — APPROVED** [review]

**Pull request merged**